app/src/main/java/com/casadelfuego/omnikiller/ktx/StringExtensions.kt

```diff
@@ -0,0 +1,18 @@
+package com.casadelfuego.omnikiller.ktx
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+fun String.toBoolean() = !isFalse()
+
+fun String.isFalse() = isBlank() || this == "0" || "false".equals(this, true)
+
+fun String.toDate(): Date? {
+    val dateFormat = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault())
+    return dateFormat.parse(this)
```

Comment:
o que acontece se o String não for uma data de acordo com esse parser?

Reply:
💥

Reply:
I think we should discuss another solution in Sunnyvale festival